src/CltPlank.elm

```diff
@@ -1,4 +1,4 @@
-module CltPlank exposing (..)
```

Comment:
I think it's preferable to explicitly define what to export, since otherwise you would end up cluttering the namespace of any file you use the library in with a lot of variables or functions that are only relevant in this file.